app/src/main/res/values-v19/styles.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="AppTheme.Base" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
2 space indentation please

Reply:
Sure @mhutti1. I have committed the requested changes.